src/frontends/tensorflow/src/decoder_proto.cpp

```diff
@@ -122,11 +163,82 @@ ov::Any DecoderProto::get_attribute(const std::string& name) const {
                                 "' attribute is not supported.");
     }
 
-    case ::tensorflow::AttrValue::ValueCase::kTensor:
-        FRONT_END_GENERAL_CHECK(false,
-                                "Conversion from Tensorflow to OpenVINO data type failed: Tensor type for '",
-                                name,
-                                "' attribute is not supported.");
+    case ::tensorflow::AttrValue::ValueCase::kTensor: {
+        const auto& tensor_proto = attrs[0].tensor();
+        const auto& tf_shape = tensor_proto.tensor_shape();
+        ov::PartialShape pshape;
+        for (int i = 0; i < tf_shape.dim_size(); i++) {
+            pshape.push_back(tf_shape.dim(i).size());
+        }
+        FRONT_END_GENERAL_CHECK(pshape.is_static(), "Dynamic shapes are not supported for Tensor attribute.");
+        const auto& tf_type = tensor_proto.dtype();
+        FRONT_END_GENERAL_CHECK(
+            TYPE_MAP().count(tf_type),
+            "Encountered unknown element type " + DataType_Name(tf_type) + " on an empty tensor_proto");
+        auto ov_type = TYPE_MAP().at(tf_type);
+        ov::Tensor res(ov_type, pshape.get_shape());
+        auto tensor_content = tensor_proto.tensor_content();
+        if (!tensor_content.empty() && tensor_proto.has_tensor_shape()) {
```

Comment:
tensor_proto.has_tensor_shape() looks a bit odd here because we already used tensor_proto.tensor_shape(); above.
I guess we need to check this at the beginning of the function